posthog/api/organization.py

```diff
@@ -130,11 +130,7 @@ def get_permissions(self) -> List[permissions.BasePermission]:
 
     def destroy(self, request, *args, **kwargs):
         instance = self.get_object()
-        for member in instance.members.all():
-            if member.organizations.count() <= 1:
-                raise exceptions.ValidationError(
-                    f"Cannot remove organization since that would leave member {member.email} organization-less, which is not supported yet."
-                )
+        instance.teams.all().delete()
```

Comment:
This will be handled automatically by Django's collector because the FK is set to `CASCADE`. I think it actually would be better to use `AnalyticsDestroyModelMixin` here so we report when the organization is deleted, and you don't need the method override.

Reply:
Not exactly handled automatically, this was failing without the additional `.delete()`. But I'll look into `AnalyticsDestroyModelMixin`.

Reply:
Done.